website/components/Hero.tsx

```
@@ -10,77 +13,131 @@ const platformLogos = [
-                            href="/activate"
-                            className="glow-btn inline-flex items-center gap-2 rounded-2xl bg-gradient-to-r from-blue-600 to-blue-500 px-10 py-4 text-base font-semibold text-white transition-all hover:from-blue-500 hover:to-cyan-500 md:px-14 md:py-5 md:text-lg"
-                        >
-                            Get Started
-                            <ArrowRight className="h-4 w-4" />
+                    <motion.div variants={itemVariants} className="flex flex-col items-center gap-6">
+                        <Link href="/activate" passHref legacyBehavior>
+                            <motion.a
+                                whileHover={{ scale: 1.02 }}
+                                whileTap={{ scale: 0.98 }}
+                                className="glow-btn inline-flex items-center gap-2 rounded-2xl bg-gradient-to-r from-[#0047ff] to-[#1a5cff] px-10 py-4 text-base font-bold tracking-wide text-white transition-colors hover:from-[#1a5cff] hover:to-[#00f0ff] md:px-14 md:py-5 md:text-lg"
+                            >
```

Biome (2.4.4)

[error] Provide a href attribute for the a element.

(lint/a11y/useValidAnchor)